doc/source/whatsnew/v1.2.0.rst

```diff
@@ -782,3 +782,5 @@ Other
 
 Contributors
 ~~~~~~~~~~~~
+
+.. contributors:: v1.1.4..v1.2.0|HEAD
```

Comment:
prob need to change this later :->

Reply:
yep, adding this now so that if we do 1.2.0rc0 before 1.1.5 and publish the 1.2.0rc0 docs, contributors get shown.

doing 1.2.0rc0 b4 1.1.5 is doable, but I think we should get 1.1.5 out before 1.2.0

Reply:
yes absolutely agree (then shouldn't this say 1.1.5 then?

Reply:
ohi c, ok then this is fine

Reply:
without a v1.1.5 tag, doing this now would fail web and docs checks